.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
we want what is in rubocop_todo.yml in here and checked in. We need to investigate if we want to make any changes to anything in there.